### DISCUSSION TITLE
Port to Godot Engine

### DISCUSSION THREAD

**Discussion by Ilnazz** [Live-Charts/LiveCharts2#1290] [Q&A]
Hello. Thanks for the amazing library.
I want to make a port of LiveCharts2 to Godot game engine.
I started porting the main platform-specific classes using examples for existing platforms, but there are points that need to be discussed. Can you help me?

**Comment by beto-rodriguez** [comment]
Sure, Let me know where I can help.

**Comment by Ilnazz** [comment]
Good afternoon.
I have completed porting Live Charts 2 to Godot.
Link to the repository:
https://github.com/Ilnazz/LiveCharts2_Godot/

![](https://github.com/Ilnazz/LiveCharts2_Godot/blob/master/Example.gif)

**Reply by beto-rodriguez** [reply]
That looks amazing, let me check that out!

**Reply by Ilnazz** [reply]
Good time of day. I would like to know if there are plans to include this port for Godot in the source code of your library?
